Packages/vcs/Lib/vcs2vtk.py

```diff
@@ -15,6 +15,14 @@
 f = open(os.path.join(vcs.prefix,"share","vcs","wmo_symbols.json"))
 wmo = json.load(f)
 
+def isTrue(x):
+    """Test if an unsanitized user input is 'true'."""
+    # I'm just encapsulating existing logic here, but this seems to miss some
+    # other cases users may enter. What about:
+    #    - Non-zero integers other than 1?
+    #    - "yes", "Yes", "Y"?
+    return x in ("y", 1, True)
```

Comment:
@dlonie labels are checked at:
https://github.com/UV-CDAT/uvcdat/blob/master/Packages/vcs/Lib/isoline.py#L540
Looks like it is speciafically checking  for "Yes/No", I think it's ok to check for Yes/No/True/False
LEt's replace with:
https://github.com/UV-CDAT/uvcdat/blob/master/Packages/vcs/Lib/VCS_validation_functions.py#L360
And mke it set to True/False rather than "y"/"n'